website/blog/2023-09-22-7.23.0.md

```diff
@@ -0,0 +1,201 @@
+
+## Highlights
+
+### Decorator Metadata ([#15895](https://github.com/babel/babel/pull/15895))
+
+The [decorator metadata](https://github.com/tc39/proposal-decorator-metadata/) Stage 3 proposal extends the [Decorator](https://github.com/tc39/proposal-decorators/) proposal, allowing decorators to communicate with each other by sharing some metadata.
+
+```javascript
+function write(_, context) {
+  context.metadata.message = "Hello from writer!";
+}
+function read(_, context) {
+  console.log(context.metadata.message);
+}
+
+@read // Logs "Hello from writer!"
+class A {
+  @write x;
+}
+```
+
+While we usually have one plugin per proposal, these two proposals are so closely related that we decided to include support for Decorator Metadata directly in `@babel/plugin-proposal-decorators`. If you are on the latest version of Babel and you are already using standard-track decorators, this proposal is enabled by default.
+
+### Source Phase Imports for WebAssembly modules ([#15829](https://github.com/babel/babel/pull/15829), [#15870](https://github.com/babel/babel/pull/15870))
+
+The [Source Phase Import](https://github.com/tc39/proposal-source-phase-imports) Stage 3 proposal allows importing an object representation of the _source_ of a module, without evaluating it and without loading its dependencies. You can load module sources using the `import source` syntax, for languages that have a defined source representation:
+
+```javascript
+import source fooSource from "foo";
+```
+
+The motivating use case is to have static syntax to load `WebAssembly.Module` objects, rather than being forced to do so dynamically:
+
+```javascript
+// Before
+const url = import.meta.resolve("./my-wasm/module.wasm");
+const responsePromise = fetch(url);
+const mod = await WebAssembly.compileStreaming(responsePromise);
+
+// After
+import source mod from "./my-wasm/module.wasm";
+```
+
+Babel now supports transforming source imports for WebAssembly modules targeting Node.js, Web-compatible environments (browsers and Deno), or both, depending on your configured [`targets`](https://babeljs.io/docs/options#targets).
+
+You can transform it using the `@babel/plugin-proposal-import-wasm-source` plugin:
+
+```json
+{
+  "targets": ["chrome 90", "firefox 90", "node 20.6"],
+  "plugins": ["@babel/plugin-proposal-import-wasm-source"]
+}
+```
+
+### Deferred Import Evaluation ([#15845](https://github.com/babel/babel/pull/15845), [#15878](https://github.com/babel/babel/pull/15878))
+
+The [Deferred Import Evaluation](https://github.com/tc39/proposal-defer-import-eval/) Stage 2 proposal allows deferring the (synchronous) evaluation of imported modules until when they are used. This can have greatly improve startup performance of your code, in case of modules with a significant initialization cost that aren't immediately used.
```

Comment:
```suggestion
The [Deferred Import Evaluation](https://github.com/tc39/proposal-defer-import-eval/) Stage 2 proposal allows deferring the (synchronous) evaluation of imported modules until they are used. This can greatly improve the startup performance of your code when using modules that have a significant initialization cost.
```